### PR TITLE
[runtime] Fix incorrect format specifier in warning message

### PR DESCRIPTION
**Pull request by valeriyvan**

### CODE REVIEW

File: stdlib/public/runtime/EnvironmentVariables.cpp [hunk 1]
```diff
@@ -115,7 +115,7 @@ static uint32_t parse_uint32_t(const char *name,
   }
   if (n > UINT32_MAX) {
     swift::warning(RuntimeErrorFlagNone,
-                   "Warning: %s=%s out of bounds, clamping to %d.\n",
+                   "Warning: %s=%s out of bounds, clamping to %u.\n",
```
Comment (CodaFi) [inline stdlib/public/runtime/EnvironmentVariables.cpp:118]: Can we use `PRIu32` from `inttypes.h` here?

Comment (valeriyvan) [reply stdlib/public/runtime/EnvironmentVariables.cpp]: done

**Review by CodaFi — APPROVED** [review]

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Comment by valeriyvan** [comment]
ping

**Comment by valeriyvan** [comment]
ping

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test macOS

**Comment by AnthonyLatsis** [comment]
@valeriyvan Please consider adding a "stdlib" prefix to the commit message. Also, "format specifier" is better; "format" by itself is ambiguous.

**Comment by valeriyvan** [comment]
> @valeriyvan Please consider adding a "runtime" prefix to the commit message (the style is up to you). Also, "format specifier" is better; "format" by itself is ambiguous.

done

**Comment by AnthonyLatsis** [comment]
@swift-ci please smoke test

**Pull request merged**